src/cascadia/TerminalSettingsModel/TerminalSettings.cpp

```diff
@@ -287,6 +287,8 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
             const til::color colorRef{ profile.TabColor().Value() };
             _TabColor = static_cast<winrt::Microsoft::Terminal::Core::Color>(colorRef);
         }
+        std::unordered_map<hstring, uint32_t> featMap{ {L"subs", 1} };

```

Comment:
(This line is just here for testing, when the json-parsing part is implemented this line will be removed and we will construct the map from the value in the profile instead)